src/task.dart

```diff
@@ -0,0 +1,13 @@
+class Task {
+  int id;
+  String title;
+  String description;
+  bool iscompleted;
```

Comment:
use spell checker extension
```suggestion
  bool isCompleted;
```